docker/Dockerfile

```diff
@@ -40,22 +42,22 @@ EXPOSE 2036 8080
 
 LABEL org.label-schema.build-date="${BUILD_TIMESTAMP}" \
   org.label-schema.vcs-ref="${GIT_REVISION}" \
-  org.label-schema.version="${VERSION}" \
+  org.label-schema.version="${TAG}" \
   org.label-schema.license="Apache-2.0" \
-  org.label-schema.name="Snow Owl OSS" \
+  org.label-schema.name="${PRODUCT_NAME}" \
   org.label-schema.schema-version="1.0" \
-  org.label-schema.url="https://github.com/b2ihealthcare/snow-owl" \
```

Comment:
Set the old default values to the arguments, so anyone who comes to this repo can build the image with just `docker build .`.
I know Nexus is something that's still required, we will remove that dependency in the next iteration of the docker build process.

Reply:
Default values are set for arguments so the image can be built as before

Reply:
Yes, but the default values are in the `Jenkinsfile`, right?

Reply:
I see them now, okay.